Packet++/src/DnsLayer.cpp

```diff
@@ -561,7 +561,7 @@ DnsResource* DnsLayer::addResource(DnsResourceType resType, const std::string& n
 DnsQuery* DnsLayer::addQuery(const std::string& name, DnsType dnsType, DnsClass dnsClass)
 {
 	// create new query on temporary buffer
-	uint8_t newQueryRawData[256];
+	uint8_t newQueryRawData[4096];
```

Comment:
isn't DNSKey a resource record and not a query?
Are there queries that exceed 256 bytes?

Reply:
DNSKey  is an answer record, here maybe needn't change to 4096. I will try revert it.